src/providers/twitch/eventsub/Connection.cpp

```diff
@@ -188,6 +188,10 @@ void Connection::onChannelModerate(
 
             if constexpr (CanMakeModMessage<Action>)
             {
+                if (getSettings()->hideDeletionActions)
+                {
+                    return;
+                }
```

Comment:
This will hide all moderation actions. At the very least, this should check for a deletion action. However, I think we concluded that we still add the messages but don't show them (like with "hide moderator actions"). Given that it's implemented as "don't add the message" in IRC, I'd be fine with a check & early out here but with a FIXME above.

Reply:
FIXME added in [`e65aa5b` (#6233)](https://github.com/Chatterino/chatterino2/pull/6233/commits/e65aa5ba0ddd43f20c238c91a5355e9236319241)